src/api/util/prettifiers.js

```diff
@@ -0,0 +1,569 @@
+    case 'SA':
+      return 'Saudi Arabia';
+    case 'SB':
+      return 'Solomon Islands';
+    case 'SC':
+      return 'Seychelles';
+    case 'SD':
+      return 'Sudan';
+    case 'SE':
+      return 'Sweden';
+    case 'SG':
+      return 'Singapore';
+    case 'SH':
+      return 'Saint Helena, Ascension and Tristan da Cunha';
+    case 'SI':
+      return 'Slovenia';
+    case 'SJ':
+      return 'Svalbard and Jan Mayen';
+    case 'SK':
+      return 'Slovakia';
+    case 'SL':
+      return 'Sierra Leone';
+    case 'SM':
+      return 'San Marino';
+    case 'SN':
+      return 'Senegal';
+    case 'SO':
+      return 'Somalia';
+    case 'SR':
+      return 'Suriname';
+    case 'SS':
+      return 'South Sudan';
+    case 'ST':
+      return 'Sao Tome and Principe';
+    case 'SV':
+      return 'El Salvador';
+    case 'SX':
+      return 'Sint Maarten (Dutch part)';
+    case 'SY':
+      return 'Syrian Arab Republic';
+    case 'SZ':
+      return 'Swaziland';
+    case 'TC':
+      return 'Turks and Caicos Islands';
+    case 'TD':
+      return 'Chad';
+    case 'TF':
+      return 'French Southern Territories';
+    case 'TG':
+      return 'Togo';
+    case 'TH':
+      return 'Thailand';
+    case 'TJ':
+      return 'Tajikistan';
+    case 'TK':
+      return 'Tokelau';
+    case 'TL':
+      return 'Timor-Leste';
+    case 'TM':
+      return 'Turkmenistan';
+    case 'TN':
+      return 'Tunisia';
+    case 'TO':
+      return 'Tonga';
+    case 'TR':
+      return 'Turkey';
+    case 'TT':
+      return 'Trinidad and Tobago';
+    case 'TV':
+      return 'Tuvalu';
+    case 'TW':
+      return 'Chinese Taipei';
+    case 'TZ':
+      return 'Tanzania, United Republic of';
+    case 'UA':
+      return 'Ukraine';
+    case 'UG':
+      return 'Uganda';
+    case 'UM':
+      return 'United States Minor Outlying Islands';
+    case 'US':
+      return 'United States';
+    case 'UY':
+      return 'Uruguay';
+    case 'UZ':
+      return 'Uzbekistan';
+    case 'VA':
+      return 'Holy See (Vatican City State)';
+    case 'VC':
+      return 'Saint Vincent and the Grenadines';
+    case 'VE':
+      return 'Venezuela, Bolivarian Republic of';
+    case 'VG':
+      return 'Virgin Islands, British';
+    case 'VI':
+      return 'Virgin Islands, U.S.';
+    case 'VN':
+      return 'Viet Nam';
+    case 'VU':
+      return 'Vanuatu';
+    case 'WF':
+      return 'Wallis and Futuna';
+    case 'WS':
+      return 'Samoa';
+    case 'YE':
+      return 'Yemen';
+    case 'YT':
+      return 'Mayotte';
+    case 'ZA':
+      return 'South Africa';
+    case 'ZM':
+      return 'Zambia';
+    case 'AA':
+      return 'User defined';
+    case 'XZ':
+      return 'International waters';
+    case 'QO':
+      return 'Oceania';
+    case 'ZZ':
+      return 'Unknown or invalid (used internally at GBIF for int. orgs)';
+    default:
+      return name;
+  }
+};
+
+export const prettifyUserType = name => {
```

Comment:
The translation file is a better place for these I would think?

Reply:
Yes, sure. I haven't fixed it yet after our discussion.